src/modeler/AutoRest.Swagger/Validation/ValidFormats.cs

```diff
@@ -22,6 +22,18 @@ public class ValidFormats : TypedRule<string>
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
         /// <summary>
+        /// What kind of open api document type this rule should be applied to
+        /// </summary>
+        public override ServiceDefinitionDocumentType ServiceDefinitionDocumentType => ServiceDefinitionDocumentType.ARM;
```

Comment:
what does autorest do if non supported format is provided ? If it blows up then we need ARM | Data-plane, I guess